src/Sarif.Viewer.VisualStudio.Core/Views/Locations.xaml

```diff
@@ -184,9 +184,14 @@
                                                                     <Setter Property="Foreground"
                                                                             Value="{DynamicResource {x:Static vs_shell:CommonDocumentColors.ListItemTextBrushKey}}"/>
                                                                 </DataTrigger>
+                                                                <DataTrigger Binding="{Binding IsSelectedNode}"
```

Comment:
>IsSelectedNode [](http://example.com/codeflow?start=95&length=14)

whats the difference between `IsSelected` and `IsSelectedNode` here?

Reply:
IsSelected indicates the location was navigated to. IsSelectedNode means the TreeViewItem is selected.